htdocs/PHP/internal_libraries/page_templates/footer.php

```diff
@@ -20,13 +20,16 @@
             echo "Español";
           } if ($_SESSION['l'] == "it") {
             echo "Italiano";
+          } if ($_SESSION['l'] == "fr") {
+            echo "Français";
           }
           ?>
         </button>
         <ul class="dropdown-menu" aria-label="<?=$translation["footer_language"];?>">
           <li><a class="dropdown-item" href="index.php?l=en">English</a></li>
           <li><a class="dropdown-item" href="index.php?l=es">Español</a></li>
           <li><a class="dropdown-item" href="index.php?l=it">Italiano</a></li>
+          <li><a class="dropdown-item" href="index.php?l=fr">Français</a></li>
```

Comment:
This one can be moved above the Italian one and that's because this list is ordered by alphabetical order.